### ISSUE TITLE
Incorrect scoping rules for `NamedExpr` in comprehensions

### ISSUE THREAD

**Issue by BarrensZeppelin** [dflook/python-minifier#119]
The following code

```python3
def f(arg, /):
  print([x for y in range(10) if (x := y // 2) & 1])
  print(arg, arg)

f(10)
```
which prints
```
[1, 1, 3, 3]
10 10
```

minifies to

```python3
A=print
def B(B):A([B for A in range(10)if(B:=A//2)&1]);A(B,B)
B(10)
```
which prints
```
[1, 1, 3, 3]
4 4
```

I think the problem is that the minifier incorrectly thinks that the variable assigned to in the assignment expression inside the comprehension is local to the comprehension scope, which is not the case.

**Comment by dflook** [comment]
Thanks for creating an issue @BarrensZeppelin, I believe this is fixed in version 2.11.3

**Comment by BarrensZeppelin** [comment]
Thanks for the quick fix. 😀
